### DISCUSSION TITLE
AI大模型3新方向--不會寫程式的人也能開發網頁？

### DISCUSSION THREAD

**Discussion by Tina-Peng-1566** [1101-GS4719-C/Python-Community-GS4719-C#2064] [General]
。Google老董：大模型3新方向，不必再認真學Python？
https://www.gvm.com.tw/article/115126

黃仁勳曾在公開演講時提到:「我們從指令驅動的計算（instruction-driven computing），也就是寫程式，轉變為意圖驅動的計算（intention-driven computing）。」
曾任Google執行長與董事長的施密特也觀察到另一重要趨勢:以文字轉行動（text to action）描述:將使用者寫下的文字轉換為程式碼，讓沒學過程式的人，也能開發網頁或應用程式。
我覺得，這就是我們現在正在做的事，而且這項能力已經變得越來越重要，尤其是對社會組的學生來說。

最近也常在IG上看到，推薦給大學生好用的生成式AI軟體，論文摘要、以文生圖、做簡報等各種功能的都有，可見現在與AI剛推出時相比，能接受「在生活中使用AI」的程度已經大幅提升，也有老師非常樂意將AI用於課堂中。

在未來，也不是說不必學Python，但至少不必擔心因不會寫程式而被社會潮流淘汰，只要學會看得懂部分程式碼、知道如何應用在生活中，並且保持著創新的思維，和對未來趨勢的眼光，相信一定能走出一條不一樣的路。

**Comment by ds0w0** [comment]
我認為，AI的進步確實在降低學習的門檻，讓我們能更輕鬆地接觸到一些原本需要專業知識的領域。像施密特提到的「文字轉行動」技術，讓沒學過程式的人也能開發應用程式，這無疑是個令人興奮的趨勢。然而，這並不代表我們可以不用認真學習。

即使AI能幫助我們解決許多問題，掌握基本的程式知識仍然是必要的。了解背後的邏輯和原理，有助於我們更有效地使用這些工具，而不只是依賴它們的表面功能。而且，學習本身也是一種提升思維能力的過程，能夠讓我們在面對未來的變化時，更加從容不迫。

AI固然是個強大的輔助工具，能夠拓寬我們的學習和創新空間，但它並不能替代我們對知識的深入理解與探索。所以，儘管AI降低了某些門檻，我們仍然應該保持對學習的認真態度，這樣才能真正把握住未來的機會。

**Comment by hunteryang0108** [comment]
我認為不必寫程式語言這個說法，僅限於那些本來就不依賴程式語言、但能使用程式語言改善效率的工作。本身就是以程式開發作為專業技術的那些工作，並沒有那麼容易的就被AI取代。不過AI模型的發展確實能讓辦公室工作以更有效率的方式進行，未來真的非常有可能從LowCode時代慢慢進入到NoCode時代。
